src/Root.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
+import { App } from './App';
+import { PeopleProvider } from './store/PeopleContext';
+import { HomePage } from './Pages/HomePage';
+import { PeoplePage } from './components/PeoplePage';
```

Comment:
The import path for `PeoplePage` should be `../components/PeoplePage` instead of `./components/PeoplePage` to match the file structure.